services/analytics-service/test-api.sh

```
@@ -0,0 +1,77 @@
+
+BASE_URL="http://localhost:9004"
+
+echo "=========================================="
+echo "Analytics Service API Test"
+echo "=========================================="
+echo ""
+
+# Test UUIDs (thay đổi theo dữ liệu thực tế của bạn)
+TEST_EXAM_ID="550e8400-e29b-41d4-a716-446655440000"
+TEST_USER_ID="550e8400-e29b-41d4-a716-446655440001"
+TEST_SUBMISSION_ID="550e8400-e29b-41d4-a716-446655440002"
```

Shellcheck (0.11.0)

[warning] TEST_SUBMISSION_ID appears unused. Verify use (or export if used externally).

(SC2034)